src/commands/CommandFlag.h

```diff
@@ -123,4 +126,6 @@ struct AUDACITY_DLL_API RegisteredMenuItemEnabler{
 // Unnecessary #include to indicate otherwise hidden link dependencies
 #include "Menus.h"
 
+static ReservedCommandFlag::Init sInitReservedCommandFlags;
```

Comment:
Could this be a potential problem?  As there could be possible multiple instances of these being created, each belonging to different translation units.